tools/SourceKit/tools/sourcekitd/include/sourcekitd/RawData.h

```diff
@@ -0,0 +1,24 @@
+//===--- RawData.h - =-------------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_SOURCEKITD_RAW_DATA_H
+#define LLVM_SOURCEKITD_RAW_DATA_H
+
+#include "sourcekitd/Internal.h"
+
+namespace sourcekitd {
+
+VariantFunctions *getVariantFunctionsForData();
```

Comment:
getVariantFunctionsForData -> getVariantFunctionsForRawData, is slightly better for clarity.